qa_forum/views.py

```diff
@@ -0,0 +1,36 @@
+import json
+from django.core import serializers
+from django.http import HttpResponse
+from django.shortcuts import render, redirect
+from django.views.decorators.csrf import csrf_exempt
+from qa_forum.forms import QuestionForm
+from qa_forum.models import Question, Answer
+
+
+def index(request):
```

Comment:
@chuckcwh It's better to have different routes to handle different actions. Here you have one method handling two possible actions. I understand that you can make a POST or GET request to the same URL and expect different actions, but it's better if you separate them into two methods (I think you can do that by routing the URL to different controller actions, correct me if I am wrong). You don't want to have a fat controller. When your project grows, it will quickly become unreadable and hard to maintain. For the very least, I would have call two separate private methods depending on whether it's POST or a GET request. (e.g. a private method called `create` to handle the question creation and a private method called something else to render the form) It is also easier to do unit tests if you break things into small chunks.